Lib/venv/__init__.py

```diff
@@ -303,8 +303,11 @@ def setup_python(self, context):
             copier(context.executable, path)
             if not os.path.islink(path):
                 os.chmod(path, 0o755)
-            for suffix in ('python', 'python3',
-                           f'python3.{sys.version_info[1]}'):
+
+            suffixes = ['python', 'python3', f'python3.{sys.version_info[1]}']
+            if sys.version_info[:2] == (3, 14):
+                suffixes.append('pythonπ')
```

Comment:
```suggestion
                suffixes.append('pythonPi')
```
?

In my console (linux framebuffer), your wariants look like "pythonп" (where pi looks like cyrillic letter Pe.

Reply:
I'm not surprised some fonts render them similarly, IIRC cyrillic Pe is directly descended from π

Reply:
> ?
> 
> In my console (linux framebuffer), your variant look like "pythonп" (where pi looks like cyrillic letter Pe.

yes! Using the original greek characters is really weird. This is what I want to express.

Reply:
The point of this easter egg is the π character, so I don't think changing to `Pi` makes sense. Marking as resolved.